src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -412,16 +408,8 @@ private MobileAdsDataProvider() {
         {
             "project43",
             "SyntheticTests/ProvidersChain",
-            "InstantGlobals.wgAdDriverRubiconFastlaneProviderCountries=[]",
-            AdsContent.MOBILETOP_LB,
-            "DirectGptMobile; RemnantGptMobile"
-        },
-        {
-            "project43",
-            "SyntheticTests/ProvidersChain",
-            "InstantGlobals.wgAdDriverRubiconFastlaneProviderCountries=[XX]",
-            AdsContent.MOBILETOP_LB,
-            "DirectGptMobile; RemnantGptMobile; RubiconFastlane"
+            AdsContent.MOBILE_TOP_LB,
+            "DirectGptMobile"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "DirectGptMobile" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)